packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -179,14 +179,7 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
   }
 
   /**
-   * @param activeUrl
-   * @param webex -.
-   * @param destination -.
-   * @param direction -.
-   * @param deviceId -.
-   * @param serverCb
-   * @param deleteCb
-   * @param indicator - Service Indicator.
+   * @ignore
```

Comment:
?

Reply:
It's a constructor. And we are not exposing classes to documentation as the developer won't create instances of these classes manually. Instances of these classes are create by SDK only. We are however providing documentation for the interfaces like ICall, ILine, etc.